source/getting-started/naming-conventions.md

```diff
@@ -91,11 +91,13 @@ to the template.
 Here's an example:
 
 ```app/routes/favorites.js
+import ajax from 'ic-ajax';
+
 export default Ember.Route.extend({
-  model: function() {
+  model() {
     // the model is an Array of all of the posts
     // fetched from this url
-    return $.ajax('/a/service/url/where/posts/live');
+    return ajax('/a/service/url/where/posts/live');
```

Comment:
This is a thing I'm not familiar with - you don't need to prepend `$.` to `ajax`? What is it called on?